README.md

```diff
@@ -15,9 +15,46 @@ NCSA Customizations for xdmod
 
 ## Description
 
+Setup a host to be a xdmod client (host which generates report files and transfers them to an xdmod server), or setup a host to be an xdmod server
 
 ## Setup
 
+### xdmod Client Setup
+
+1. Identify the host to be the xdmod client, requirements are:
+    - Host can run `sacct` and `scontrol show config` commands
+    - Host can reach the xdmod server over the network
+1. Generate an SSH key for the client, this can be generated anywhere for now (does not have to be generated from the xdmod client host)
+    - If the project uses only RHEL 8 or newer servers, use an ed25519 key type
+    - If the project needs to support older OS, use a rsa key type
+1. Save SSH key contents as encrypted eyaml in your projects control repo, see [link](https://wiki.ncsa.illinois.edu/display/PUPPET/Encrypting+Puppet+hiera+with+eyaml#EncryptingPuppethierawitheyaml-EncryptingHieraValueswitheyaml)
+    - Typically save this to your project repos `data/secrets/common.eyaml`
+    - The specific values will look like this:
+    ```yml
+    profile_xdmod::client::ssh_key_priv: >
+      #LONG ENCRYPTED STRING
+    profile_xdmod::client::ssh_key_pub: >
+      #LONG ENCRYPTED STRING
+    ```
+1. Setup xdmod specific values, fine to put these in data/common.yaml
+    ```yml
+    profile_xdmod::client::ssh_key_type: "ssh-ed25519"
+    profile_xdmod::client::xdmod_hostname: "FQDN-OF-XDMOD-SERVER-HERE"
+    profile_xdmod::client::xdmod_service_acct: "USERNAME-OF-XDMOD-SERVICE-ACCOUNT"
+    ```
+1. Include the client class for whichever host will be your client (site-modules/role/manifests/YOUR-CLASS-HERE.pp:)
+    ```yml
+    include ::profile_xdmod::client
+    ```
+1. On the project repo that controls the xdmod server, add the client to the clients hash:
+    ```yml
+    profile_xdmod::server::clients:
+      "FQDN-OF-XDMOD-CLIENT":
+        hostname: "FQDN-OF-XDMOD-CLIENT"
```

Comment:
This seems redundant. I understand why, but just pointing out it would be nice to only spec the client FQDN once.

Reply:
This is fixed, no longer need to put hostname twice